src/ServiceControlInstaller.Engine/Instances/ServiceControlNewInstance.cs

```diff
@@ -50,8 +50,8 @@ public void AddRemoteInstance(string apiUri)
             }
         }
 
-        [XmlElement(typeof(XmlTimeSpan))]
-        public TimeSpan AuditRetentionPeriod { get; set; }
+        [XmlElement(typeof(XmlNullableTimeSpan))]
+        public TimeSpan? AuditRetentionPeriod { get; set; }
```

Comment:
I wonder if we still need this.

New instances do not require an Audit Retention Period

Reply:
I was trying to remove it but then the interface inheritance war hit me hard 😉